Lib/test/_test_multiprocessing.py

```diff
@@ -653,6 +653,47 @@ def test_forkserver_sigkill(self):
             self.check_forkserver_death(signal.SIGKILL)
 
 
+class TestStdOutAndErr(unittest.TestCase):
+    @staticmethod
+    def closeIO(stream_name):
+        getattr(sys, stream_name).close()
+
+    @staticmethod
+    def removeIO(stream_name):
+        setattr(sys, stream_name, None)
+
+    def test_closed_stdio(self):
+        """
+        bpo-28326: multiprocessing.Process depends on sys.stdout being open
+        """
+        self.run_process(self.closeIO)
+
+    def test_no_stdio(self):
+        """
+        bpo-31804: If you start Python by pythonw then sys.stdout and
```

Comment:
I think there's no need for the docstring (or comment) to be that long. Just put the reference to the issue number + a 2- or 3-line summary.

Reply:
I trimmed the docstring.